builder/swig/java.bzl

```diff
@@ -45,8 +45,14 @@ def _swig_java_wrapper_impl(ctx):
         wrap_src = ctx.actions.declare_file("{}_wrap.c".format(module_name))
         swig_headers = []
 
+    includes = []
+    for f in ctx.files.includes:
+        include = ctx.actions.declare_file(f.short_path)
+        _copy_to_bin(ctx, f, include)
+        includes.append(include)
```

Comment:
Note: we could also copy none of the .i and .swgs to bazel-bin, but in that case SWIG cannot find the generated header instead.